gradle.properties

```diff
@@ -10,50 +10,50 @@ researchgateReleaseVersion=2.8.0
 testngVersion=7.6.1
 ballerinaGradlePluginVersion=2.0.1
 
-ballerinaLangVersion=2201.9.0
+ballerinaLangVersion=2201.10.0-20240806-083400-aabac46a
 
-stdlibSqlVersion=1.13.0
+stdlibSqlVersion=1.13.2
```

Comment:
Need to update the new timestamped version right?

Reply:
yeah, updated with the new timestamp version